arviz/plots/parallelplot.py

```diff
@@ -18,6 +20,7 @@ def plot_parallel(
     colord="C1",
     shadend=0.025,
     ax=None,
+    norm_method="normal",
```

Comment:
What about removing `normalize` argument and setting `norm_method = None` by default.